libtransmission/torrent-files.cc

```diff
@@ -218,13 +218,10 @@ bool tr_torrent_files::move(
     {
         auto const remove_empty_directories = [](char const* filename)
         {
-            if (isEmptyDirectory(filename))
```

Comment:
Will `remove_empty_directories` now remove directories that aren't empty?

Reply:
@ile6695 Tbh I’m not sure…
In what situation is it desirable to keep non empty folders. why would you ever need to keep some files in the original location?

Reply:
hmm, ok.
ill have a play around tonight

Reply:
@ckerr @ile6695 ive re-added,
the fix still works with the **isEmptyDirectory** check